foreman/requirements.txt

```diff
@@ -7,11 +7,16 @@
 amqp==2.1.4               # via kombu
 billiard==3.5.0.2         # via celery
 celery==4.0.2
-django==1.10.6
+django==1.11
```

Comment:
Hmm, whoops. I don't think I have a reason to be doing that. I think I may have run into a bug that was introduced in `1.11` though so that's why most of the project is `1.10.6`. I'll change this to match that.